x-pack/legacy/plugins/maps/public/layers/vector_layer.js

```diff
@@ -621,30 +630,37 @@ export class VectorLayer extends AbstractLayer {
     const pointLayer = mbMap.getLayer(pointLayerId);
     const symbolLayer = mbMap.getLayer(symbolLayerId);
 
-    let mbLayerId;
+    let markerLayerId;
+    let textLayerId;
     if (this._style.arePointsSymbolizedAsCircles()) {
```

Comment:
Could you add a comment? This seems really esoteric, how layer-ids are assigned (ie. mapbox-gl quirck with multiple symbol-layers).